app/templates/admin/admin_base.html

```diff
@@ -0,0 +1,83 @@
+{% import 'admin/layout.html' as layout with context -%}
+{% import 'admin/static.html' as admin_static with context %}
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>{% block title %}{% if admin_view.category %}{{ admin_view.category }} - {% endif %}{{ admin_view.name }} - {{ admin_view.admin.name }}{% endblock %}</title>
+    {% block head_meta %}
+        <meta charset="UTF-8">
```

Comment:
We seem to have some weird additional indentation for the addition head child elements here.

Reply:
Ah I copied this from Flask Admin. We can maybe redo the the whole template in time.

Reply:
We can revisit at a later date! 😁🤜